cli/cmd/generate.go

```diff
@@ -515,6 +522,15 @@ func parseGenerateFlags(cmd *cobra.Command) (*generateFlags, error) {
 	}, nil
 }
 
+// getDefaultManifest returns the default manifest without reference values, or
+// it includes the default reference values for AKS.
+func getDefaultManifest(aksReferenceValues bool) manifest.Manifest {
```

Comment:
This will need to change as soon as we introduce another set of default reference values. I'd just inline this into `runGenerate`.